src/styles/main.scss

```diff
@@ -119,6 +120,25 @@ h1 {
   margin: 0 8px 0 0;
 }
 
+.current-score-animation {
+  position: absolute;
+  transform: translateY(10px);
+  font-size: 20px;
+  color: #684e33;
+  animation: jump .5s ease-in;
+}
+
+@keyframes jump {
+  0% {
+    transform: translateY(10px);
+    visibility: 1;
```

Comment:
The 'visibility' property is being set to 1. This property can only accept the values 'visible', 'hidden', 'collapse', and 'initial'. If you want to change the visibility, use the appropriate value.